### ISSUE TITLE
Change ECS service to only have one task running during update

### ISSUE THREAD

**Issue by JimLiu0** [HS-Tools/HS-BG-Leaderboards-Bot#32]
Currently the behavior of updating a task is: for 1-2 minutes the old task and the new task will be running until the old task is deleted.

I would like the behavior to be: until the new task is ready to be run, run the old task and then kill it once the new task is ready.

This may not be possible however.

**Comment by JimLiu0** [comment]
Not relevant anymore now that I'm back to using rpi
